components/checkbox/overview.md

```diff
@@ -0,0 +1,275 @@
+---
+title: Overview
+page_title: Checkbox Component for Blazor Overview
+description: Overview of the Checkbox for Blazor
+slug: components-checkbox-overview
+tags: telerik,blazor,checkbox,overview
+published: True
+position: 0
+---
+
+# Checkbox Overview
+
+The Checkbox component allows you to add more customizable checkboxes to your Blazor application. It maintains the behavior of the standard HTML checkbox and provides checked, unchecked and indeterminate states.
+
+To use a Telerik Checkbox for Blazor
+
+1. add the `TelerikCheckBox` tag
+1. provide `Value` (one-way data binding) or `bind-Value` (two-way data binding) property
+1. (optional) set the `Indeterminate` or `bind-Indeterminate` property
+1. (optional) set the `ValueChanged` and `IndeterminateChanged` (if using one-way data binding).
+
+This article contains information about:
+* [Features](#features)
+* [Events](#events)
+* [Examples](#examples)
+
+>caption Basic setup of the Telerik CheckBox using two-way data binding
+
+````CSHTML
+@*Basic setup of the Telerik CheckBox Component*@
+<TelerikCheckBox Id="myCheckBox"
+                 @bind-Value="isSelected">
+</TelerikCheckBox>
+<label for="myCheckBox">@(isSelected ? "Selected" : "Not selected")</label>
+
+@code {
+        private bool isSelected {get; set;}
+}
+````
+
+
+## Features
+
+The CheckBox provides the following features:
+
+* `Class` - the CSS class that will be rendered on the main wrapping element of the CheckBox.
+* `Enabled` - whether the component is enabled.
+* `Id` - renders as the `id` attribute on the `<select />` element, so you can attach a `<label for="">` to it.
+* `TabIndex` - the `tabindex` attribute rendered on the CheckBox.
+* `Value` and `bind-Value`- mapped to the `Checked` property of the normal HTML checkbox
+  * The `Value` and `bind-Value` accept `bool` type
```

Comment:
This should work with `bool?` as well and that should be documented. Unless it does not, of course.